### ISSUE TITLE
DHCPv6 Prefix Delegation over PPPoE

### ISSUE THREAD

**Issue by GroverChouT** [openSUSE/wicked#877]
A few ISPs now support IPv6 PD over PPPoE but currently with wicked is unable to make it work if not using a custom script. My ISP issues a /64 prefix on the PPPoE interface by Router Advertisement. If a DHCPv6 DP request is sent, it will return a /56 prefix for other use. Usually for other devices in the home.

I tried combining `BOOTPROTO=dhcp6` with `PPPMODE` like this:

```
BOOTPROTO='dhcp6'
STARTMODE='auto'
PPPMODE='pppoe'
DEVICE='enp2s0.20'
PROVIDER='provider0'
ZONE=external
DHCLIENT6_MODE='prefix'
```

An error will occur:

```
Aug 09 02:40:36 box wickedd-nanny[937]: device ppp0: call to org.opensuse.Network.Addrconf.ipv6.dhcp.requestLease() failed: Object does not support requested method
```

Another thought is to call another DHCP client with a script, and set `POST_UP_SCRIPT` and `PRE_DOWN_SCRIPT` might be the solution to this.

But since wicked already has a DHCP client built-in, I think I should post a feature request here.

**Comment by mtomaschewski** [comment]
Yes, but as I didn't had the time to test prefix delegation with PPPoE, it's not yet enabled.
See https://github.com/openSUSE/wicked/blob/master/src/dhcp6/device.c#L1669

**Comment by bertofurth** [comment]
Any chance you could post the sample script and other required setup to workaround this issue until this issue is resolved?
